public/app/percona/entitlements/components/PageContent/PageContent.test.tsx

```diff
@@ -0,0 +1,34 @@
+import { mount } from 'enzyme';
+import { dataTestId } from '@percona/platform-core';
+import { PageContent } from './PageContent';
+import React from 'react';
+
+describe('PageContent', () => {
+  it('should display the noData section when no data is passed', async () => {
+    const wrapper = mount(<PageContent hasData={false} emptyMessage="empty" />);
```

Comment:
Can we use RTL instead? We've moving away from Enzyme and Iaroslavna is working on migrating all our tests as well

Reply:
Changed to RTL

Reply:
👍